CDCgov/prime-simplereport
# Terraform pagerduty support

## Related Issue or Background Info

- CDCGov/prime-devops#2 requires us to have a pagerduty integration exposed to our existing alerts systems

## Changes Proposed

- update the metrics alerts to take multiple action IDs instead of just one (so we can send events to multiple receivers)
- update the `global` package to set up terraform-managed pagerduty integrations to Production and Demo services

## To verify:

* terraform best practices
* is this how we want to use our existing Pagerduty services? Should we add new ones instead? Should we import the existing integrations so we don't have multiple integration webhooks for Azure?
* do we want to add any customization to the data as it flows through these webhooks?

## Additional Information

A follow-up PR will make use of these changes in the seven (7) deployed API environments.

## Screenshots / Demos

## Checklist for Author and Reviewer

### UI
- N/A Any changes to the UI/UX have been approved by design 
- N/A Any new or edited error messages have been approved by design 

### Testing

### Changes are Backwards Compatible
- N/A Database changes are submitted as a separate PR
- N/A GraphQL schema changes are backward compatible with older version of the front-end

### Security
- N/A Changes with security implications have been approved by a security engineer (changes to  authentication, encryption, handling of PII, etc.)

## Cloud
- LOL DevOps team has been notified if PR requires ops support
